src/lib/util.ts

```diff
@@ -12,15 +13,19 @@ export default class Util {
 
   public static fromWei(amount: BigNumber): number {
     const web3 = new Web3();
-    return Number(web3.fromWei(amount, "ether"))
+    return Number(web3.utils.fromWei(amount, "ether"))
   }
 
   // TODO: should probably return a BigNumber instead of a string.
-  public static toWei(amount: number): string {
+  public static toWeiAsString(amount: number): string {
     const web3 = new Web3();
     return web3.toWei(amount, "ether")
   }
 
+  public static toWei(amount: number): BigNumber {
+    return new BigNumber(arc.web3.utils.toWei(amount, "ether"))
```

Comment:
we use arc.web3 here but create a new Web() object above. would be good to be consistent